core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/SearchResourceRequestBuilder.java

```diff
@@ -42,6 +42,10 @@
 	
 	private int limit = 50;
 	
+	private int minTermMatch = 0;
```

Comment:
What does the default value `0` mean? How can I ensure my search does not use this feature? What happens with existing searches that use this generic API?

Reply:
Fixed this. To enable the new feature this parameter must be configured.